### ISSUE TITLE
Referencing multiple apiModules in NgModule

### ISSUE THREAD

**Issue by fireappsAdmin** [cyclosproject/ng-swagger-gen#87]
I have multiple configs up and running with no issues. I do have a question though, I feel like i'm missing something obvious. In each config file I have apiModule set to true.

Is there a way to change the name of the apiModule that is imported in NgModule? I have several APIs that I want to import and right now they are all named apiModule. 

Any help is appreciated. Thanks!

**Comment by luisfpg** [comment]
Indeed, it would be nice to add a prefix in the configuration file, so both `ApiModule` and `ApiConfiguration` would be prefixed with that.
For example, if prefix is `Customers`, the names would be `CustomersApiModule` and `CustomersApiConfiguration`.

**Comment by remkoboschker** [comment]
How would you add a particular api to a lazily loaded feature module?
